docs/fastdds/discovery/static.rst

```diff
@@ -280,3 +280,28 @@ Or you can specify the STATIC EDP configuration content directly.
 |    :end-before: //!                                  |
 |    :dedent: 8                                        |
 +------------------------------------------------------+
+
+Checking STATIC EDP XML Files
```

Comment:
I suggest moving this section before `STATIC EDP XML Example`. I think it is more logical explaining that there is a way to check the XML validity before showing an example.

Reply:
Moved in [af7f5b8](https://github.com/eProsima/Fast-DDS-docs/pull/330/commits/af7f5b8485581c6a8ec1bc626569cbdb8e7f9f1d)